src/main/java/net/mcreator/ui/modgui/TagGUI.java

```diff
@@ -70,11 +68,12 @@ public TagGUI(MCreator mcreator, ModElement modElement, boolean editingMode) {
 		JPanel pane3 = new JPanel(new BorderLayout());
 		pane3.setOpaque(false);
 
-		items = new MCItemListField(mcreator, ElementUtil::loadBlocksAndItems);
-		blocks = new MCItemListField(mcreator, ElementUtil::loadBlocks);
+		items = new MCItemListField(mcreator, ElementUtil::loadBlocksAndItems, false, true);
+		blocks = new MCItemListField(mcreator, ElementUtil::loadBlocks, false, true);
+		entities = new SpawnableEntityListField(mcreator, true);
+		biomes = new BiomeListField(mcreator, true);
 		functions = new ModElementListField(mcreator, ModElementType.FUNCTION);
 		entities = new SpawnableEntityListField(mcreator);

```

Comment:
This is now defined twice.

Reply:
Well, thanks auto-merge hehe. Good catch, will fix